src/components/pages/admin/callouts/callouts.interface.ts

```diff
@@ -39,6 +39,7 @@ export interface TitleAndImageStepProps {
 export interface SettingsStepProps {
   whoCanTakePart: 'members' | 'everyone';
   allowAnonymousResponses: 'none' | 'guests' | 'all';
+  requireCaptcha: 'none' | 'guest' | 'all';
```

Comment:
Does it make sense to move the enum from the backend to common and use it here? Then it only needs to be used in one place

Reply:
It does, I was just being lazy :upside_down_face:. I will fix